stubs/protobuf/METADATA.toml

```diff
@@ -1,2 +1,2 @@
 version = "3.20.*"
```

Comment:
this wants to update to `4.21.*` but we're waiting a bit for a new version of mypy to release